share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -329,12 +329,8 @@ macro(rocm_setup_license HEADER_ONLY)
                 "rocm-cmake warning: Multiple license files found, "
                 "please specify one using CPACK_RESOURCE_FILE_LICENSE."
             )
-        elseif(_num_licenses EQUAL 0)
-            message(AUTHOR_WARNING
-                "rocm-cmake warning: Could not find a license file, "
-                "please specify one using CPACK_RESOURCE_FILE_LICENSE."
-            )
-        else()
```

Comment:
We should keep this warning, and just add a LICENSE file to the tests.

Reply:
Warning reinstated, and placeholder LICENSE files have been added to all tests which use rocm_create_package.